test/test-ncu.js

```diff
@@ -93,6 +94,17 @@ describe('npm-check-updates', function () {
                 });
         });
 
+        it('should fall back to package.json search when receiving empty content on stdin', function (done) {
+            var childProcess = child.exec('node bin/ncu', function (error, stdout) {
+                if (error) {
+                    throw new Error(error);
```

Comment:
Throwing an error in a callback will not necessarily bubble up. You should call `done(error)` instead to explicitly fail the test.